KeychainMinderTests/KeychainMinderTests.m

```diff
@@ -151,28 +141,26 @@ - (void)testRequirementsString {
   XCTAssertTrue([selfCS validateWithRequirement:xctestRequirements]);
 }
 
-//
+
 // Used to debug the XPC Connection. In KeychainMinderAgent.m be sure to change the requirement
 // string to com.apple.xctest.
-//
-//- (void)testXPCConnection {
-//  NSXPCConnection *connectionToService =
-//  [[NSXPCConnection alloc] initWithMachServiceName:kKeychainMinderAgentMachServiceName
-//                                           options:NSXPCConnectionPrivileged];
-//  connectionToService.remoteObjectInterface = [NSXPCInterface interfaceWithProtocol:
-//                                               @protocol(KeychainMinderAgentProtocol)];
-//  [connectionToService resume];
-//
-//  id remoteObject = [connectionToService
-//                     remoteObjectProxyWithErrorHandler:^(NSError *error) {
-//                       NSLog(@"%@", [error debugDescription]);
-//                     }];
-//  NSData *data = [NSKeyedArchiver archivedDataWithRootObject:@"TOMTOM"];
-//  [remoteObject setPassword:data withReply:^(BOOL reply) {
-//    NSLog(@"setPassword [%hhd]", reply);
-//  }];
-//
-//  sleep(100000);
-//}
+- (void)testXPCConnection {
+  NSData *passwordData = [NSKeyedArchiver archivedDataWithRootObject:@"TOMTOM"];
+
+  NSXPCConnection *connectionToService =
+  [[NSXPCConnection alloc] initWithMachServiceName:kKeychainMinderAgentServiceName
+                                           options:NSXPCConnectionPrivileged];
+  connectionToService.remoteObjectInterface = [NSXPCInterface interfaceWithProtocol:
+                                               @protocol(KeychainMinderAgentProtocol)];
+  [connectionToService resume];
+
+  id remoteObject = [connectionToService remoteObjectProxyWithErrorHandler:^(NSError *error) {
+    NSLog(@"%@", [error debugDescription]);
+  }];
+
+  [remoteObject setPassword:passwordData withReply:^(BOOL reply) {
+    XCTAssertTrue(reply);
```

Comment:
This should use XCTestExpectation. See [here](https://github.com/google/santa/blob/master/Tests/LogicTests/SNTFileWatcherTest.m#L54) for an example